systemvm/debian/opt/cloud/bin/setup/cloud-early-config

```diff
@@ -89,6 +89,12 @@ start() {
   rm -f /root/.rnd
   echo "" > /root/.ssh/known_hosts
 
+  if which growpart > /dev/null; then
+    growpart /dev/vda 2
```

Comment:
I checked the 4.15 systemvmtemplate, growpart is not available but resize2fs is available. Can we use fdisk instead of growpart to resize partition instead, or some other alternative? This would save from creating a new systemvmtemplate as we get closer to the 4.15 release?

Reply:
@rhtyd root partition must be the last partition (it is linux swap for now), so this feature requires new systemvm template. let's move to 4.16.
I will work on other hypervisors except kvm.